sandboxes/staticanalysis/staticanalyze.go

```diff
@@ -0,0 +1,235 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/ossf/package-analysis/internal/log"
+	"github.com/ossf/package-analysis/internal/pkgecosystem"
+	"github.com/ossf/package-analysis/internal/staticanalysis"
+	"github.com/ossf/package-analysis/internal/staticanalysis/obfuscation"
+	"github.com/ossf/package-analysis/internal/utils"
+	"github.com/ossf/package-analysis/internal/worker"
+)
+
+var (
+	ecosystem   = flag.String("ecosystem", "", "Package ecosystem (required)")
+	packageName = flag.String("package", "", "Package name (required)")
+	version     = flag.String("version", "", "Package version (ignored if local file is specified)")
+	localFile   = flag.String("local", "", "Local package archive containing package to be analysed. Name must match -package argument")
+	help        = flag.Bool("help", false, "Prints this help and list of available analyses")
+	analyses    = utils.CommaSeparatedFlags("analyses", "obfuscation", "comma-separated list of static analyses to perform")
+)
+
+const defaultJsParserPath = "internal/staticanalysis/parsing/js/babel-parser.js"
+
+func getJsParserPath() (string, error) {
+	parserPath := defaultJsParserPath
+	parserType := "default"
+
+	customParserPath := os.Getenv("JS_PARSER")
```

Comment:
nit: can this be an argument passed to the binary, rather than an env var?

Reply:
For this, I was thinking that inside the sandbox image, everything will have set paths anyway, so the default path can be used most of the time. The other thing is that there will be a separate parser for every language, so it might end up complicating/polluting the list of arguments if we have a CLI flag for every parser. I can still do it though. 

Alternatively we could have a config file, but that seems like a can of worms. Env vars seemed like a reasonable option given that they're also used by the worker.

Reply:
Will resolve in another PR